src/cow-react/modules/swap/services/ethFlow/index.ts

```diff
@@ -0,0 +1,53 @@
+import { EthFlowContext } from '@cow/modules/swap/services/swapFlow/types'
+import { swapFlowAnalytics } from '@cow/modules/swap/services/common/steps/analytics'
+import { signEthFlowOrderStep } from '@cow/modules/swap/services/ethFlow/steps/signEthFlowOrderStep'
+import confirmPriceImpactWithoutFee from 'components/swap/confirmPriceImpactWithoutFee'
+import { logSwapFlow } from '@cow/modules/swap/services/utils/logger'
+import { getSwapErrorMessage } from '@cow/modules/swap/services/common/steps/swapErrorHelper'
+import { PriceImpact } from 'hooks/usePriceImpact'
+import { addPendingEthFLowOrderStep } from '@cow/modules/swap/services/ethFlow/steps/addPendingEthFLowOrderStep'
+
+export async function ethFlow(input: EthFlowContext, priceImpactParams: PriceImpact) {
+  logSwapFlow('ETH FLOW', 'STEP 1: confirm price impact')
+  if (priceImpactParams?.priceImpact && !confirmPriceImpactWithoutFee(priceImpactParams.priceImpact)) return
+
+  logSwapFlow('ETH FLOW', 'STEP 2: send transaction')
+  // TODO: check
+  swapFlowAnalytics.swap(input.swapFlowAnalyticsContext)
```

Comment:
should we use the same swapFlow analytics or create a unique one for eth-flow?

Reply:
@MindyCoW  would know best